pageserve/static/style.css

```diff
@@ -0,0 +1,48 @@
+/* Add a black background color to the top navigation */
+.topnav {
```

Comment:
So this navbar CSS styling was taken pretty directly from w3 schools' examples - is there any concern there?

Reply:
If it works, it works. Personally, I really passionately hate CSS `float`, where something like [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is much cleaner.

Reply:
Changed to use flexbox in 91c9612

Reply:
If the actual text was copied (as opposed to simply used as an example), you should check the license. If you actually wrote it yourself based on w3 schools information, then it's your code to license as you see fit (and include a license header at the top).

Note that w3schools has an explicit license for fair use (with examples) but not an OSS copyright license:

https://www.w3schools.com/about/about_copyright.asp